core/services/head_tracker_test.go

```diff
@@ -20,40 +20,55 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+func firstHead(t *testing.T, store *strpkg.Store) models.Head {
+	h := models.Head{}
+	if err := store.GetRawDB().Order("number asc").First(&h).Error; err != nil {
```

Comment:
Do we need GetRawDB here? Why do we need to skip the lock?

Reply:
Why would we need a global lock for this query?